nbdime/merging/autoresolve.py

```diff
@@ -414,12 +431,13 @@ def autoresolve_decision_on_dict(dec, base, sub, strategies):
     if strategy is not None:
         decs = strategy2action_dict(sub, le, re, strategy, subpath, dec)
     elif le.op == DiffOp.PATCH and re.op == DiffOp.PATCH:
+        assert False
```

Comment:
Are we thinking alike if I say this will never happen because patches have previously been popped up to the common path?